llama/evo_llama.py

```diff
@@ -10,11 +10,21 @@
 import mlx.nn as nn
 from mlx.utils import tree_unflatten
 
-SCALE = np.random.choice([0.025, 0.05, 0.1, 0.2])
+SCALE = np.random.choice([0.025, 0.05, 0.1, 0.2, 0.2, 0.2])
 cache_size = 0
 cache_modifier = None
 
 
+def hallucinator():
+    person = np.random.choice(["Olivier Teytaud", "Jean Raoul", "Gym Beckoul", "Sebastien Tortolini", "Gardio Pateoni", "Jean Patou", "Stephane Teytaud", "Simon Simbad", "Jean Volant", "John Pool", "Shi-Jim Chen", "Karl Teytaud", "Olivier Teytaut"])
+    activity = np.random.choice(["krav-maga", "karate", "the game of Go", "tae-kwon-do", "sumo", "table tennis", "number theory", "haikus", "judo", "chess", "chinese checkers", "soccer", "theoretical physics", "chaos theory", "Montessori studies", "modern surrealism", "Mangas", "jazz", "ballet dancing", "opera", "hard rock", "speed metal", "hip hop", "breakdancing", "rap", "world music", "fusion cooking"])
+    appreciate = np.random.choice(["I love", "I am interested in", "I like", "I study", "I have recently learnt about", "I'd like to know more about", "I need information about", "Can you provide information about"])
+    question = np.random.choice(["Do you know", "Do you have information about", "Do you know something about"])
+    return appreciate + " " + activity + ". " + question + " " + person + " ?", "Does the sentence << ", " >> imply that " + person + " is good at " + activity + " ?"
+
```

Comment:
Maybe it could be clearer with an f-string: f"{appreciate} {activity}. {question} ?"